src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -133,7 +133,7 @@ private void validateTrustDetails(OverseasEntitySubmissionDto overseasEntitySubm
 
     public Errors validatePartial(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
         if (isRoeUpdateEnabled && overseasEntitySubmissionDto.isForUpdate()) {
-            validatePartialUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
+            // validatePartialUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
```

Comment:
Why comment out here ?

Reply:
This was commented out to fix breaking change and can't be enabled. There's a ticket to fix the validation UAR-711.